brainglobe_atlasapi/atlas_generation/atlas_scripts/perens_stereotaxic_mri_mouse.py

```diff
@@ -0,0 +1,402 @@
+__version__ = "3"
```

Comment:
Our versioning is (currently) independent from the atlas version, so this will now make version 1.0.

```suggestion
__version__ = "0"
```